public/static/app/perf-bb/hazardLevelCell.js

```diff
@@ -7,19 +7,22 @@ mciModule.directive('hazardLevelCell', function() {
   }
 
   function ratio(a, b) {
-    return 100 * (a > b ? a / b - 1 : -b / a + 1)
+    return 100 * ((a/b) - 1);
   }
 
   function renderUnfoldedItem(scope, row) {
-    var a = scope.row.entity.statistics.next.mean
-    var b = scope.row.entity.statistics.previous.mean
+    var a = scope.row.entity.statistics.next.mean;
+    var b = scope.row.entity.statistics.previous.mean;
+    scope.ratio = ratio(a, b);
     // Check mode (ops/sec or latency)
```

Comment:
nit: can you change the comment to something like '// a < 0 is a latency change, ensure ratio correct. '